.golangci.yml

```diff
@@ -0,0 +1,32 @@
+# golangci-lint configuration used for CI
+run:
+  tests: true
+  timeout: 15m
+  skip-dirs-use-default: true
+
+linters-settings:
+  goimports:
+    local-prefixes: github.com/vmware-tanzu/nsx-operator
+  revive:
+    ignore-generated-header: false
+    severity: warning
+    confidence: 0.8
+    rules:
+      - name: unreachable-code
+      - name: errorf
+      - name: range
+      - name: superfluous-else
+      - name: var-declaration
+      - name: duplicated-imports
+
+linters:
+  disable-all: true
+  enable:
+    - misspell
+    - gofmt
```

Comment:
seemly, gofumpt is btter than gofmt?

Reply:
I see for other open source Golang project, they use gofmt in .golangci.yml. Just keep the same.